### DISCUSSION TITLE
Inline radio buttons under the label

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1843] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello,
according to [documentation](https://fomantic-ui.com/collections/form.html#radio-checkbox) we have two layout for radio buttons:

![download](https://user-images.githubusercontent.com/5399265/105046979-0dd49180-5a6a-11eb-8661-6f50ef4a5f65.png)

I need to have label and radio buttons displayed in the following way:

![download2](https://user-images.githubusercontent.com/5399265/105047025-1fb63480-5a6a-11eb-9e65-b78351a0dcd1.png)

How to do that using Fomantic classes?

**Comment by lubber-de** [comment]
If you click on the empty tag to the right, you will see the necessary html code
![image](https://user-images.githubusercontent.com/18379884/105048290-ac152700-5a6b-11eb-826b-3e9556cde05b.png)
which will reveil that it's done via [inline fields](https://fomantic-ui.com/collections/form.html#inline-field)

```html
<div class="ui form">
  <div class="inline fields">
    <label for="fruit">Select your favorite fruit:</label>
    <div class="field">
      <div class="ui radio checkbox checked">
        <input type="radio" name="fruit" checked="" tabindex="0" class="hidden">
        <label>Apples</label>
      </div>
    </div>
    <div class="field">
      <div class="ui radio checkbox">
        <input type="radio" name="fruit" tabindex="0" class="hidden">
        <label>Oranges</label>
      </div>
    </div>
    <div class="field">
      <div class="ui radio checkbox">
        <input type="radio" name="fruit" tabindex="0" class="hidden">
        <label>Pears</label>
      </div>
    </div>
    <div class="field">
      <div class="ui radio checkbox">
        <input type="radio" name="fruit" tabindex="0" class="hidden">
        <label>Grapefruit</label>
      </div>
    </div>
  </div>
</div>
```

**Comment by smartm0use** [comment]
Maybe you misunderstand me.
I can see two ways of displaying radio buttons:

1) Inline radio buttons after the label
2) Vertical list of radio buttons under the label

I need to have a third way:
3) Inline buttons under the label

**Comment by smartm0use** [comment]
Is it possible to do that using Fomantic classes?

**Comment by lubber-de** [comment]
Two options:
- Use [field widths](https://fomantic-ui.com/collections/form.html#width)
- Put all checkboxes next to each other inside a single field

https://jsfiddle.net/lubber/wn6ar20g/5/
![image](https://user-images.githubusercontent.com/18379884/105205330-6f1d6300-5b45-11eb-8766-150962b12018.png)

**Comment by ko2in** [comment, marked as answer]
I adjusted @lubber-de fiddle (https://jsfiddle.net/ko2in/y7eup8b0/2/) to get a better layout with nice spacing between the fields.

![checkboxes](https://user-images.githubusercontent.com/930315/105479006-555d5680-5cd2-11eb-881a-b764b03d4f62.png)

**Comment by smartm0use** [comment]
Thank you for your replies.
I prefer the second options by lubber-de.

What about to add that in the documentation, as third option for Fomantic radio buttons?

**Reply by ko2in** [reply]
The [documentation](https://fomantic-ui.com/collections/form.html#radio-checkbox) has enough explanation in my opinion. You just need to tweak the usages of `field`, `inline fields` according to your needs.

**Comment by smartm0use** [comment]
Anyway, @ko2in your code is different against the screenshot you attached.

**Reply by ko2in** [reply]
Not sure how it looks for you. But, it works exactly like my screenshot.

**Comment by smartm0use** [comment]
![image](https://user-images.githubusercontent.com/5399265/105738940-dc5f3880-5f37-11eb-91b2-4c9815967a91.png)

**Reply by ko2in** [reply]
The inline fields are grid layouts. It changes the width according to the screen size.
![inline-fields](https://user-images.githubusercontent.com/930315/105748546-af7b4780-5f70-11eb-84f0-ddc195a80978.gif)

**Comment by smartm0use** [comment]
Sorry, my bad!
